docs/sql-guide/functions/function-datetimefromparts.md

```diff
@@ -0,0 +1,55 @@
+---
+title: DATETIMEFROMPARTS()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+
+# DATETIMEFROMPARTS() function
+
+The `DATETIMEFROMPARTS()` function takes integer date and time arguments and returns timestamp.
+
+## Syntax
+
+```
+DATETIMEFROMPARTS(int_year, int_month, int_date, int_hour, int_minute, int_seconds, int_milliseconds)
```

Comment:
why not just say "year", "month", etc?

The data types in the table below are integers.
Plus the example has integers.